cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -1099,7 +1106,7 @@ func newServer(conf rootConfig) (*Server, error) {
 		return nil, fmt.Errorf("failed to setup TMPDIR under %s: %v", conf.index, err)
 	}
 
-	if conf.dbg {
+	if srcLogLevelIsDebug() {
```

Comment:
instead of doing this, can we ask the sglog what level it is?

Reply:
As far as I can tell this isn't an option but I defer to @bobheadxi if there is a way to do this. The level is established on the zap core logger (both the global logger on init and any logger instances created afterwards) and we don't appear to have access to the zap logger (I can only view it while debugging)

Reply:
Once we migrate all logs to sourcegraph/log then I believe we can remove this function